testing/__init__.py

```diff
@@ -50,14 +50,17 @@ def child_pids(pid):
     """Return a list of direct child PIDs for the given PID."""
     children = set()
     for p in LocalPath('/proc').listdir():
-        stat = p.join('stat')
-        if stat.isfile():
-            stat = stat.open().read()
+        try:
+            stat = open(p.join('stat').strpath).read()
```

Comment:
cannot deal with the 5+ different `py.error` exceptions that occur if we let `py.path` do the `open` (and which don't inherit from `IOError`)